interfaces/builtin/custom_device.go

```diff
@@ -76,7 +76,7 @@ func (iface *customDeviceInterface) validateFilePath(path string, attrName strin
 		return fmt.Errorf(`custom-device %q path is not clean: %q`, attrName, path)
 	}
 
-	if _, err := utils.NewPathPattern(path); err != nil {
+	if _, err := utils.NewPathPattern(path, true); err != nil {
```

Comment:
when we rarely use boolean args we use constants to help reading:

const allowCommas = true
... utils.NewPathPattern(path, allowCommas) ...

Reply:
this needs a corresponding unit test